src/Microsoft.AspNetCore.Mvc.Core/Internal/ApiBehaviorApplicationModelProvider.cs

```diff
@@ -57,6 +57,13 @@ public void OnProvidersExecuting(ApplicationModelProviderContext context)
             foreach (var controllerModel in context.Result.Controllers)
             {
                 var isApiController = controllerModel.Attributes.OfType<IApiBehaviorMetadata>().Any();
+                if (isApiController &&
+                    controllerModel.ApiExplorer.IsVisible == null)
```

Comment:
Is `ApiExplorer` ever null?

Reply:
Nope: https://github.com/aspnet/Mvc/blob/dev/src/Microsoft.AspNetCore.Mvc.Core/ApplicationModels/ControllerModel.cs#L34